localstack/utils/bootstrap.py

```diff
@@ -733,6 +756,25 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, traceback):
         self.shutdown()
 
+    def ip_address(self, docker_network: str | None = None) -> str:
+        """
+        Get the IP address of the container
+
+        Optionally specify the docker network
+        """
+        # TODO: podman may not have this information
+        inspect_result = self.container_client.inspect_container(self.id)
+        if docker_network is None:
+            return list(inspect_result["NetworkSettings"]["Networks"].values())[0]["IPAddress"]
+        else:
+            for network, details in inspect_result["NetworkSettings"]["Networks"].items():
+                if network == docker_network:
+                    return details["IPAddress"]
```

Comment:
In the container client, there is `get_container_ip` and `get_container_ip_for_network` already. We should use that utilities here instead